Src/BootCamp.Chapter/Checks.cs

```diff
@@ -16,25 +16,25 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
+            Program.AskingAndConvertingINT();
             return 0;
```

Comment:
All tests are failing because you are not reusing the returns of Program.AskingAndConverting... functions. Return the result instead of default values (0, "", etc)

Reply:
Owww, yeah, I am stupid, thanks!